lib/rspec/matchers/built_in/include.rb

```diff
@@ -5,8 +5,11 @@ module BuiltIn
       # Provides the implementation for `include`.
       # Not intended to be instantiated directly.
       class Include < BaseMatcher
-        def initialize(*expected)
-          @expected = expected
+        # @private
+        attr_reader :expecteds
```

Comment:
Actually I think this is ok, it's not expectations because it's expected values but it's ambiguous as to what these are